### DISCUSSION TITLE
How to create image cropping and zoomable component

### DISCUSSION THREAD

**Discussion by MemphisEx** [MemphisEx/Shopping_MERN#2] [Announcements]
Hi, members!
I need to create the image cropping and zoomable component using react.
Because it needs to upload the avatar in the user profile.
Who has the similar experience in react?

**Comment by VladaWebDev** [comment]
Well, you can use the react-easy-crop, or react-image-crop.
https://www.npmjs.com/package/react-easy-crop
https://www.npmjs.com/package/react-image-crop

**Comment by MemphisEx** [comment]
Thanks, @VladaWebDev 
I will try to use it.
